src/components/WriteEditModal/ImageInput.tsx

```diff
@@ -1,53 +1,47 @@
 'use client';
 
-import React, { useState } from 'react';
+import { useState, ChangeEvent } from 'react';
 import Image from 'next/image';
-import Carmera from '@/public/svgs/camera.svg';
-import DeleteImage from '@/public/svgs/deleteImage.svg';
+import CarmeraIcon from '@/public/svgs/camera.svg';
+import DeleteImageIcon from '@/public/svgs/deleteImage.svg';
 import classNames from 'classnames/bind';
 import styles from './ImageInput.module.scss';
 
-export default function ImageInput() {
-  const cn = classNames.bind(styles);
+const cn = classNames.bind(styles);
 
+export default function ImageInput() {
   const [selectedImageFile, setSelectedImageFile] = useState<string[]>([]);
-  const handleChangeImage = (e: React.ChangeEvent<HTMLInputElement>) => {
-    if (selectedImageFile.length === 4) {
+  const handleChangeImage = (e: ChangeEvent<HTMLInputElement>) => {
+    const { files } = e.target;
+    if (selectedImageFile.length >= 4 || !files) {
       alert('이미지 4개 이상 불가');
       return;
     }
-    if (e.target.files) {
-      const file = e.target.files[0];
-      const imageUrl = URL.createObjectURL(file);
-      setSelectedImageFile([...selectedImageFile, imageUrl]);
-    }
+    const imageUrl = URL.createObjectURL(files[0]);
+    setSelectedImageFile((prev) => [...prev, imageUrl]);
   };
   const handleClickDeleteImage = (clickedImageIndex: number) => {
-    const updatedImageFile = [
-      ...selectedImageFile.slice(0, clickedImageIndex),
-      ...selectedImageFile.slice(clickedImageIndex + 1),
-    ];
-    setSelectedImageFile(updatedImageFile);
+    setSelectedImageFile((prev) => [...prev.slice(0, clickedImageIndex), ...prev.slice(clickedImageIndex + 1)]);
   };
   return (
     <div className={cn('container')}>
       <div className={cn('title-div')}>
-        <h1 id={cn('title-h1')}>사진/동영상 첨부</h1>
-        <h3 id={cn('sub-title-h3')}>최대 4장</h3>
+        <h1 id={cn('title')}>사진/동영상 첨부</h1>
+        <h3 id={cn('sub-title')}>최대 4장</h3>
       </div>
       <div>
         <form className={cn('input-div')}>
           {selectedImageFile.length >= 4 || (
-            <label htmlFor="imageInput" className={cn('label-input-div', `${selectedImageFile && 'width-quater'}`)}>
-              <Carmera />
+            <label htmlFor="imageInput" className={cn('label-input', `${selectedImageFile && 'width-quater'}`)}>
+              <CarmeraIcon />
```

Comment:
정확하지는 않은데 이런식으로 가능했던거같아요

```suggestion
            <label htmlFor="imageInput" className={cn('label-input', {width-quater: selectedImageFile })}>
              <CarmeraIcon />
```